src/main/java/CommandLine/Parser.java

```diff
@@ -0,0 +1,106 @@
+package CommandLine;
+
+import Task.TaskList;
+import Task.Task;
+
+import static java.lang.Integer.parseInt;
+
+public class Parser {
+    public Parser() {
+
+    };
+
+    /**
+     * Parses the input supplied by user, adds it to the task list if possible
+     * This method helps to parse inputs, calling the appropriate commands where possible
+     * @param  taskList the current taskList
+     * @param  s the string that was parsed
+     */
+    public static String parse(TaskList taskList, String s) {
+        Line line = new Line();
+        String[] parts = s.split(" ", 2);
```

Comment:
Do you think we can use a more descriptive name to improve readability, like `inputTokens`?